input/groups/weapon.json

```diff
@@ -1096,80 +928,281 @@
+        4465,
+        4529,
+        4505,
+        4474,
+        4538,
+        4380,
+        4446,
+        4383,
+        4415
       ],
-      "Legendary: Generation 2 - Experiment > Perfected > Precursor > Legendary": [
-        6276,
-        7300,
-        7206,
-        7078,
-        7337,
-        6966,
-        6717,
-        6466,
-        8389,
-        8145,
-        7907,
-        6506,
-        7024,
-        8051,
-        8576,
-        8748
+      "Dungeon: Sorrow's Embrace - Dark Asuran": [
+        4387,
+        4516,
+        4452,
+        4420,
+        4359,
+        4487,
+        4398,
+        4401,
+        4530,
+        4532,
+        4470,
+        4375,
+        4473,
+        4378,
+        4379,
+        4412,
+        4508,
+        4381,
+        4382
       ],
-      "Beaded": [
-        3904,
+      "Dungeon: Citadel of Flame - Molten": [
+        4353,
+        4481,
+        4354,
+        4355,
+        4515,
+        4486,
+        4423,
+        4426,
+        4458,
+        4395,
+        4460,
+        4492,
+        4397,
+        4528,
+        4467,
+        4376,
+        4537,
+        4377,
+        4444
+      ],
+      "Dungeon: Honor of the Waves - Kodan": [
+        4480,
+        4449,
+        4418,
+        4388,
+        4455,
+        4424,
+        4394,
+        4365,
+        4366,
+        4527,
+        4369,
+        4436,
+        4503,
+        4472,
+        4536,
+        4408,
+        4442,
+        4413,
+        4447
+      ],
+      "Dungeon: Crucible of Eternity - Inquest": [
+        4485,
+        4389,
+        4358,
+        4425,
+        4364,
+        4461,
+        4525,
+        4432,
+        4402,
+        4435,
+        4501,
+        4471,
+        4439,
+        4535,
+        4409,
+        4506,
+        4410,
+        4507,
+        4443
+      ],
+      "Dungeon: The Ruined City of Arah - Dragon's Deep": [
+        4416,
+        4448,
+        4518,
+        4391,
+        4362,
+        4491,
+        4430,
+        4494,
+        4367,
+        4434,
+        4468,
+        4469,
+        4406,
+        4440,
+        4441,
+        4476,
+        4445,
+        4509,
+        4414
+      ],
+      "Fractals of the Mists - Fractal": [
+        4519,
+        4392,
+        4521,
+        4489,
+        4393,
+        4427,
+        4462,
+        4495,
+        4466,
+        4531,
+        4500,
+        4437,
+        4405,
+        4438,
+        4539,
+        4477,
+        4478,
+        4510,
+        4479
+      ],
+      "Fractals of the Mists - Fractal > Gold Fractal": [
+        6464,
+        6369,
+        6178,
+        6373,
+        6405,
+        6439,
+        6215,
+        6443,
+        6187,
+        6155,
+        6192,
+        6384,
+        6385,
+        6194,
+        6227,
+        6323,
+        6355,
+        6358,
+        6206
+      ],
+      "Fractals of the Mists - Fractal > Gold Fractal > Abyssal Fractal": [
```

Comment:
While they obviously related, you don't technically need fractal or gold fractal weapons to get abyssal, I don't believe? Also, seems a little arbitrary for these to be tagged with Fractals of the Mists and not Nightmare or Caustic Nightmare weapons, which are in spot. Given they already have fractal in the name could drop the FotM label and just sort them in with other sets - could go either way.

Reply:
I softened the requisites for hierarchical classification a while ago: https://github.com/Lon-ami/gw2-wardrobe-unlock/commit/bb87c40f242e8e908a1dd69b18b1d2bee134d8c3 mostly for organizational purposes, since everything got mixed in with the removal of categories based on game modes.

I wouldn't give a minor set like Nightmare a prefix, but if you want one, "Fractals of the Mists: Nightmare Fractal - Nightmare" and "Fractals of the Mists: Nightmare Fractal - Nightmare > Caustic Nightmare" should be it.

If you like that prefix approach, you could also create a "Fractals of the Mists: Other" instead of the "Standalone: Fractals of the Mists", and change how the whole standalone thing is approached, at least for game modes (Dungeon, Fractals of the Mists, Raid, Strike Mission, Dragon Response Mission, Player vs. Player, World vs. World).

You would then be segmenting each category by putting game mode stuff on top, then generic stuff at the bottom. I prefer to have everything standalone/other at the bottom, and have filters as the visual cue for where each thing comes from (remember all those new ones I created), but you do you, think about it :D.